.github/workflows/maven-publish-to-nexus.yml

```
@@ -6,6 +6,11 @@ on:
       SERVICE_LOCATION:
         required: true
         type: string
+    inputs:
```

actionlint (1.7.12)

[error] key "inputs" is duplicated in "workflow_call" section. previously defined at line:5,col:5

(syntax-check)